examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -63,7 +61,7 @@ class FeederActor extends Actor {
   def receive: Receive = {
     case SubscribeReceiver(receiverActor: ActorRef) =>
       println("received subscribe from %s".format(receiverActor.toString))
-      receivers = LinkedList(receiverActor) ++ receivers
+      receivers = MutableList(receiverActor) ++ receivers
```

Comment:
I agree this is more readable but existing code was using ++ so I maintained it .
let me update it.